src/services/shims.ts

```diff
@@ -886,12 +867,9 @@ namespace ts {
         /// Emit
         public getEmitOutput(fileName: string): string {
             return this.forwardJSONCall(
-                "getEmitOutput('" + fileName + "')",
+                `getEmitOutput('${fileName}')`,
                 () => {
-                    var output = this.languageService.getEmitOutput(fileName);
-                    // Shim the API changes for 1.5 release. This should be removed once
-                    // TypeScript 1.5 has shipped.
-                    (<any>output).emitOutputStatus = output.emitSkipped ? 1 : 0;
+                    let output = this.languageService.getEmitOutput(fileName);

```

Comment:
Might as well collapse this line and the next to just `this.languageService.getEmitOutput(fileName)`